### PR TITLE
Subscriptions: fixing examples

### PR DESCRIPTION
**Pull request by prateekbh**
- Fix amp example
- Whitelist subscription attributes
- Execute action from amp-subscriptions-google

### CODE REVIEW

File: examples/amp-subscriptions.amp.html [hunk 1]
```diff
@@ -208,12 +208,7 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
         </div>
 
 
-        <div subscriptions-action="login" subscriptions-display="NOT loggedIn">Login Local</div>
-
-        <div subscriptions-display="NOT loggedIn"
-          subscriptions-action="login"
-          subscriptions-service="subscribe.google.com"
-          subscriptions-decorate>Login</div>
+        <div subscriptions-action="login" subscriptions-display="NOT loggedIn">Login</div>
```
[the remark this answers is not in the document]
Comment (prateekbh) [reply examples/amp-subscriptions.amp.html]: yep

File: src/sanitizer.js [hunk 2]
```diff
@@ -131,6 +131,8 @@ const WHITELISTED_ATTRS = [
   'subscriptions-actions',
   'subscriptions-section',
   'subscriptions-display',
+  'subscriptions-service',
+  'subscriptions-decorate',
```
[the remark this answers is not in the document]
Comment (prateekbh) [reply src/sanitizer.js]: done now

File: examples/amp-subscriptions.amp.html [hunk 3]
```diff
@@ -339,6 +334,10 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
   </template>
   <template type="amp-mustache" subscriptions-dialog subscriptions-display="NOT loggedIn">
     <div>Dialog for non logged in users<div>
+    <div subscriptions-display="NOT loggedIn"
```
[the remark this answers is not in the document]
Comment (prateekbh) [reply examples/amp-subscriptions.amp.html]: fixed

File: examples/amp-subscriptions.amp.html [hunk 4]
```diff
@@ -334,12 +329,16 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
...
   </template>
-  <template type="amp-mustache" subscriptions-dialog subscriptions-display="NOT loggedIn">
+  <template type="amp-mustache" subscriptions-dialog subscriptions-display="NOT data.loggedIn">
     <div>Dialog for non logged in users<div>
-    <div subscriptions-display="NOT subscribed">btw u are not subscribed</div>
+    <div subscriptions-display="NOT data.loggedIn"
+      subscriptions-action="subscribe"
+      subscriptions-service="subscribe.google.com"
+      subscriptions-decorate>Login</div>
+    <div subscriptions-display="NOT grantReason">btw u are not subscribed</div>
```
Comment (dvoytenko) [inline examples/amp-subscriptions.amp.html]: Let's do `grantReason != "SUBSCRIBED"` (SUBSCRIBER, whichever is the latest on this.)

Comment (prateekbh) [reply examples/amp-subscriptions.amp.html]: reverted this and loggedIn change as that stuff is in PR #15059

**Pull request merged**